### ISSUE TITLE
Error thrown for limit

### ISSUE THREAD

**Issue by ryansmith94** [kucherenko/jscpd#160]
Hey, I think there might be some issue with your type checking on command arguments. I'm seeing the error below in [our CircleCI builds for v0.6.23](https://circleci.com/gh/LearningLocker/xapi-validation/801?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

```
npm run duplication

> @learninglocker/xapi-validation@0.0.0-development duplication /root/repo
> jscpd --limit 1

ERROR: --limit expects a number. Use `jscpd --limit=NUMBER`
```

https://github.com/LearningLocker/xapi-validation/pull/63

**Comment by kucherenko** [comment]
Thank you for reporting, will fix it soon, for now I can suggest use previous version 0.6.22 or jscpd@1.0.0-alpha.2, also you can try to use limit=1.0

**Comment by ryansmith94** [comment]
Thanks @kucherenko, using "1.0" works for now.

**Comment by ryansmith94** [comment]
Sorry accidentally close this.

**Comment by ArgosBen** [comment]
+1 seeing this too

**Comment by kucherenko** [comment]
Fixed in jscpd@0.6.24
